cache_images/fedora-netavark_packaging.sh

```diff
@@ -19,6 +19,12 @@ msg "Updating/Installing repos and packages for $OS_REL_VER"
 bigto ooe.sh $SUDO dnf update -y
 
 INSTALL_PACKAGES=(\
+	btrfs-progs-devel
+	golang
+	gpgme-devel
+	libassuan-devel
+	libseccomp-devel
+	systemd-devel
```

Comment:
Brent, if you spin these again in this PR, could you please replace the `[tab]` indentation with spaces, and also sort the package list alphabetically?